src/Psl/Xml/format_error.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml;
+
+use LibXMLError;
+use Psl\Str;
+
+/**
+ * This function can be used to format a LibXMLError.
+ *
+ * @psalm-suppress MissingThrowsDocblock
+ */
+function format_error(LibXMLError $error): string
```

Comment:
I decided to add all these format functions in here.
This is currently used inside the exception, but could also be used to e.g. build an XML linter.
Don't think any of these functions should go into the internal namespace.